include/fs/littlefs.h

```diff
@@ -92,13 +121,59 @@ struct fs_littlefs {
  * @param name the name for the structure.  The defined object has
  * file scope.
  */
-#define FS_LITTLEFS_DECLARE_DEFAULT_CONFIG(name)			 \
-	FS_LITTLEFS_DECLARE_CUSTOM_CONFIG(name,				 \
-					  CONFIG_FS_LITTLEFS_READ_SIZE,	 \
-					  CONFIG_FS_LITTLEFS_PROG_SIZE,	 \
+#define FS_LITTLEFS_DECLARE_DEFAULT_CONFIG(name) \
+	FS_LITTLEFS_DECLARE_CUSTOM_CONFIG(name, \
+					  CONFIG_FS_LITTLEFS_READ_SIZE, \
+					  CONFIG_FS_LITTLEFS_PROG_SIZE, \
 					  CONFIG_FS_LITTLEFS_CACHE_SIZE, \
 					  CONFIG_FS_LITTLEFS_LOOKAHEAD_SIZE)
 
+/** @brief Define a littlefs configuration with customized size
+ * characteristics and custom operations for the actual device access.
+ *
+ * For more details see :c:macro:`FS_LITTLEFS_DECLARE_CUSTOM_CONFIG`.
+ *
+ * @param name the name for the structure.  The defined object has
+ * file scope.
+ * @param read_sz see @kconfig{CONFIG_FS_LITTLEFS_READ_SIZE}
+ * @param prog_sz see @kconfig{CONFIG_FS_LITTLEFS_PROG_SIZE}
+ * @param cache_sz see @kconfig{CONFIG_FS_LITTLEFS_CACHE_SIZE}
+ * @param lookahead_sz see @kconfig{CONFIG_FS_LITTLEFS_LOOKAHEAD_SIZE}
+ * @param read_op function pointer to read operation
+ * @param prog_op function pointer to program operation
+ * @param erase_op function pointer to erase operation
+ * @param sync_op function pointer to sync operation
+ * @param device_open_op function pointer to open device operation
+ * @param device_close_op function pointer to close device operation
+ * @param device_get_block_size_op function pointer to get device block size operation
+ * @param device_get_size_op function pointer to get device size operation
+ */
+#define FS_LITTLEFS_DECLARE_CUSTOM_OPERATIONS_CONFIG(name, read_sz, prog_sz, cache_sz, \
```

Comment:
Maybe redefine this as `FS_LITTLEFS_DECLARE_CUSTOM_CONFIG_AND_OPS`, above `FS_LITTLEFS_DECLARE_CUSTOM_CONFIG`,  and then use it in the `FS_LITTLEFS_DECLARE_CUSTOM_CONFIG` definition but fill the last four params as 0.

Reply:
done